.github/workflows/pre-merge-python.yml

```
@@ -28,7 +28,7 @@ jobs:
     strategy:
       matrix:
         framework:
-          - vllm
+          - dynamo
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)